config/routes.rb

```diff
@@ -123,6 +123,10 @@
       :as => :edit_cart_item
   # delete '/cart/empty' => 'application#empty_cart', :as => :empty_cart
   # put '/cart/update' => 'application#update_cart', :as => :update_cart
+  post '/edit_reservation_form' => 'catalog#submit_cart_updates_form',
+       :as => :submit_cart_updates_form
+  put '/change_reservation_dates' => 'catalog#change_reservation_dates',
+      :as => :change_reservation_dates
 
```

Comment:
so the reload catalog cart is used on the catalog page, but on the edit res form page, it uses the submit_cart_updates_form route. It also uses the change_reservation_dates to handle the dates in the sidebar. There was an issue with using the same function to handle the reloading on both pages since they are loading from 2 different controllers (reservation controller and catalog controller). (Sorry for taking so long I've been a bit swamped with essays and final projects these two weeks)